releases.md

```diff
@@ -0,0 +1,100 @@
+# Tekton Pipeline Releases
+
+## Release Frequency
+
+Tekton Pipelines follows the Tekton community [release policy][release-policy]
+as follows:
+
+- Versions are numbered according to semantic versioning: `vX.Y.Z`
+- A new release is produced on a monthly basis
+- Four releases a year are chosen for long term support (LTS). All remaining
+  releases are supported for approximately 1 month (until the next release is
+  produced)
+    - LTS releases take place in January, April, July and October every year
+    - The first Tekton Pipelines LTS release will be **v0.41.0** in October 2022
+    - Releases happen towards the middle of the month, between the 13th and the
+      20th, depending on week-ends and readyness
+
+Tekton Pipelines produces nightly builds, publicly available on
+`gcr.io/tekton-nightly`. 
+
+### Transition Process
+
+Before release v0.41 Tekton Pipelines has worked on the basis of an undocumented
+support period of four months, which will be maintained for the releases between
+v0.37 and v0.40.
+
+## Release Process
+
+Tekton Pipeline releases are made of YAML manifests and container images.
+Manifests are published to cloud object-storage as well as
+[GitHub][tekton-pipeline-releases]. Container images are signed by
+[Sigstore][sigstore] via [Tekton Chains][tekton-chains]; signatures can be
+verified through the [public key][chains-public-key] hosted by the Tekton Chains
+project.
+
+Further documentation available:
+
+- The Tekton Pipeline [release process][tekton-releases-docs]
+- [Installing Tekton][tekton-installation]
+- Standard for [release notes][release-notes-standards]
+
+## Releases
+
+### v0.40
```

Comment:
Should we combine this with the list of releases in README.md so we're not maintaining it in two places?

Reply:
Yes, we should do something like that.
If that's ok, once this is merged I can add links to docs and examples and link from the readme to here

Reply:
any reason not to just combine them in this PR?

Reply:
A couple of reasons:

- This PR is important for the Tetkon graduation, so I wanted to keep the PR quick and simple  
- I shared it with other projects as an example, so I would like to have as little noise as possible in it as possible
- I prefer to do one change at the time

But if you're adamant about having it in here I can add an extra commit.